src/main/java/com/reshiftsecurity/plugins/intellij/service/AnalyticsService.java

```diff
@@ -254,7 +254,8 @@ private void processActions(boolean consentExempt) {
                         .addHeader("User-Agent", this.userAgent)
                         .build();
 
-                client.newCall(request).execute();
+                Response response = client.newCall(request).execute();
+                response.body().close();
```

Comment:
Suggestion, minor: Here chain the calls if you're not using the response var any more?